tools/README.md

```diff
@@ -49,84 +49,68 @@ This is a small tool to manipulate with secret key. Along with generating new on
 
 Main usage pattern is
 ```bash
-echo <input with secret> | dscp-keygen <input-type-option> --command <command>
+echo <input with secret> | dscp-keygen <input-type-option> <command> <command-arguments>
 ```
 
-Input defines which secret key will be used as base for further operations.
-It can be specified directly, as encrypted key with password, as path to keyfile or be generated from scratch, depending on `input-type-option` option.
+Input defines which secret key will be used as a base for further operations.
+It can be specified directly, as encrypted key with a password, as a path to the keyfile or be generated from scratch, depending on `input-type-option` option.
 
-Execute `dscp-keygen --help` for the list of available 'input-type' options.
+Execute `dscp-keygen --help` for the list of available 'input-type' options and to see available commands.
 
-Command stands for data to be derived from the base secret key. Commands have options (not mandatory), which vary how data will be displayed.
+Each command stands for data to be derived from the base secret key. For the description and list of arguments for each command execute `dscp-keygen <command> --help`.
 
-List of options types:
-
-* View - determines how to display raw data, can be `raw`, `base64` or `hex`.
-* Prettiness - for large outputs, whether to use pretty multiline output.
-  Can be `pretty` or `one-line`
-* Password - password used to generate secret key.
-
-Following commands are supported:
-
-* `secret[:<view=base64>]` - display secret itself.
-* `public[:<view=hex>]` - display public key.
-* `address` - address, base58bitcoin.
-* `esecret[:<password="">[:<view=hex>]]` - display encrypted secret key.
-* `keyfile[:<password="">[:<prettiness=pretty>]]` - display keyfile.
-* `educator-auth:endpointName` - produce JWT token for educator node from given educator secret key and name of accessed endpoint.
-* `student-submission[:<seed="">]` - produce JSON request for `POST /api/student/v1/submissions` endpoint.
+Note, that name of command argument sometimes collide with options related to the input (e.g. `--password` may relate to secret on input or to secret on output).
```

Comment:
* name of *the* command 
* sometimes collide*s*
* relate to *a* secret (not sure, pls check)